var/spack/repos/builtin/packages/arrayfire/package.py

```diff
@@ -64,10 +62,28 @@ def cmake_args(self):
             self.define_from_variant('AF_BUILD_CUDA', 'cuda'),
             self.define_from_variant('AF_BUILD_FORGE', 'forge'),
             self.define_from_variant('AF_BUILD_OPENCL', 'opencl'),
+            self.define('BUILD_TESTING', self.run_tests),
         ])
+
+        if '+cuda' in self.spec:
+            arch_list = ['{}.{}'.format(arch[:-1], arch[-1])
+                                for arch in self.spec.variants['cuda_arch'].value]
```

Comment:
Same style/indentation issue here with line 70 not lining up proper with line 69.